src/extensions/BynderImage/BynderImage.js

```diff
@@ -88,9 +87,9 @@ BynderImage.propTypes = {
   sdk: PropTypes.object.isRequired
 };
 
-init(sdk => {
-  ReactDOM.render(<BynderImage sdk={sdk} />, document.getElementById("root"));
-});
+// init(sdk => {
```

Comment:
@ajwhitley why is this commented out? Will it break anything?

Reply:
Yes, it throws errors on other extensions.